pyramid_swagger/api.py

```diff
@@ -102,298 +102,6 @@ def view_for_api_declaration(request):
     return view_for_api_declaration
 
 
-def _low_level_translate(url, is_marshaling=True):
```

Comment:
I very much 👍  removal of this kind of low-level detail from pyramid-swagger. Can you add an entry in the Changelog for this improvement? I usually just make a new "in development" heading to keep track of improvements since the last release.